docs/submit/install-ospool-ap.md

```diff
@@ -177,20 +195,20 @@ the [Organizing and Submitting HTC Workloads Tutorial](https://portal.osg-htc.or
 are particularly relevant.
 
 ### Specifying a project
-OSG will only run jobs that have a registered *project* associated with them.
+OSPool resources will only run jobs that have a registered *project* associated with them.
 Users must follow the
 [instructions for starting a project in OSG-Connect](https://portal.osg-htc.org/documentation/overview/account_setup/starting-project/)
 to register a project.
 
-A project is associated with a job by adding a ProjectName line to the user's submit file.
+A project is associated with a job by adding a `+ProjectName` line to the user's submit file.
 For example:
 
 ```file
 +ProjectName = "My_Project"
 ```
 
-__The double quotes are necessary__. If not quoted, *My_Project* will be interpreted as an expression,
-and most likely evaluate to undefined, and prevent your job from running.
+__The `+` and the double quotes are necessary__.
+Otherwise, the job will fail to submit or it will not run in the OSPool.
```

Comment:
Huh, can't we address this in AP config, i.e. in the packaging?

Reply:
Sure, but this is what's implemented today.

Reply:
Mmk, that's fine for now then. Could you add it to your todo list to create a ticket for updating the osg-flock config?

Reply:
I checked the osg-connect access points and we don't have an EXTENDED_SUBMIT_COMMAND for ProjectName, only JobDurationCategory; I'd rather not put something in osg-flock that we're not even doing ourselves...